src/HttpCache/EventListener/AddHeadersListener.php

```diff
@@ -29,18 +29,22 @@ final class AddHeadersListener
     private $etag;
     private $maxAge;
     private $sharedMaxAge;
+    private $staleWhileRevalidate;
+    private $staleIfError;
     private $vary;
     private $public;
     private $resourceMetadataFactory;
```

Comment:
```suggestion
    private $vary;
    private $public;
    private $resourceMetadataFactory;
    private $staleWhileRevalidate;
    private $staleIfError;
```

Please follow the same ordering as the constructor arguments.